configator/generate_build_gradle.py

```diff
@@ -0,0 +1,23 @@
+
+""""Function to generate a gradlebuild file"""
+
+print("Let's work on getting your build.gradle file generated!")
+
+def create_gradlebuild():
+    """
+    creating a file with gatorgradle version
+    that can be read and written to
+    """
+
+#creating a variable and assigning it to strings for desired output
+    gradlebuild = """
+plugins {
+    id "org.gatored.gatorgradle" version "0.5.1"
+}
+"""
+#creating a build.gradle file
+    with open("build.gradle", "w", encoding="utf8") as generate:
```

Comment:
This will overwrite the `build.gradle` file if it exists. You should either 1) warn the user and not overwrite the file, or 2) try to insert the plugins block or `id "org.gatored.gatorgradle" version "0.5.1"` line in the correct place in that existing file, or print out instructions for the user to do that modification instead.

Reply:
This can be addressed during the next sprint.